trino/pom.xml

```diff
@@ -32,9 +32,23 @@
     <properties>
         <maven.compiler.source>17</maven.compiler.source>
         <maven.compiler.target>17</maven.compiler.target>
-        <dep.airlift.version>214</dep.airlift.version>
-        <presto.version>406</presto.version>
+        <airlift.version>214</airlift.version>
+        <trino.version>406</trino.version>
         <guava.version>31.1-jre</guava.version>
+        <hive.version>2.1.1</hive.version>
+        <airlift-units.version>1.6</airlift-units.version>
+        <airlift-slice.version>0.44</airlift-slice.version>
+        <jackson.version>2.13.1</jackson.version>
+        <guice.version>5.1.0</guice.version>
+        <inject.version>1</inject.version>
+        <jol.version>0.16</jol.version>
+        <servlet-api.version>4.0.1</servlet-api.version>
+        <commons-collections.version>3.2.1</commons-collections.version>
+        <avro.version>1.10.1</avro.version>
+        <jersey.version>1.19.4</jersey.version>
+        <guava-retrying.version>2.0.0</guava-retrying.version>
+        <commons-lang.version>2.6</commons-lang.version>
+        <commons-io.version>2.4</commons-io.version>
```

Comment:
Move to parent pom?

Reply:
I tried to move the `trino` dependencies to the parent dependency management, and left the others in the trino module pom file because of the versions may be conflict with other modules.